content/v2.0/organizations/members/add-member.md

```diff
@@ -7,25 +7,33 @@ menu:
     name: Add a member
     parent: Manage members
 weight: 201
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+<!-- ## Add a member to an organization in the InfluxDB UI -->
 
-1. Click the **Settings** tab in the navigation bar.
+<!-- 1. Click the **Settings** tab in the navigation bar. -->
 
-    {{< nav-icon "settings" >}}
+<!--     {{< nav-icon "settings" >}} -->
 
-2. Click on the name of an organization, then select the **Members** tab.
+<!-- 2. Click on the name of an organization, then select the **Members** tab. -->
 
-_Complete content coming soon_
+<!-- _Complete content coming soon_ -->
 
 ## Add a member to an organization using the influx CLI
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add) to add a user to an organization.
+You will need the ID of the user you are trying to add.
 
-_Complete content coming soon_
+1. Get a user's ID by running `influx org members list -n <org-name>`.
+
+2. Add the user with this command:
+
+    ```sh
+    influx org members add -n <org-name> -o <user-ID>
+    ```
+
+{{% cloud-msg %}}
+Adding users is currently unavailable in InfluxDB Cloud 2.
```

Comment:
- Move this to the top of the doc, just under the initial introduction.
- Adding ~users~ **members** is currently...
  "Users" and "members" are two different things. Users exist outside of organizations. A user can be added as a member of multiple organizations.